docs/csharp/language-reference/compiler-messages/cs8401.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Compiler Error CS8401
+ms.date: 07/11/2020
+f1_keywords:
+  - "CS8401"
+helpviewer_keywords:
+  - "CS8401"
+author: Youssef1313
+---
+# Compiler Error CS8401
+
+To use '@$' instead of '$@' for an interpolated verbatim string, please use language version '8.0' or greater.
+
+In older versions before C# 8.0, verbatim interpolated strings were only allowed using `$@`. Starting with C# 8.0, you can use `$@` or `@$`.
+
+## To correct this error
+
+Do one of the following options:
+
+- Replace `@$` with `$@`.
+- Upgrade to C# 8.0 or later.
```

Comment:
```suggestion
Either replace `@$` with `$@`, or upgrade to C# 8.0 or later.
```

This formatting might have made sense if there were more than two options. Instead, make it a simple sentence.